test/e2e/specs/site-editor/user-global-styles-revisions.spec.js

```diff
@@ -27,6 +27,19 @@ test.describe( 'Global styles revisions', () => {
 		await editor.canvas.click( 'body' );
 	} );
 
+	test( 'should display no revisions message if landing via command center', async ( {
+		page,
+	} ) => {
+		await page.keyboard.press( 'Meta+k' );
```

Comment:
Based on [failure traces](https://github.com/WordPress/gutenberg/actions/runs/5641131033), the command doesn't open the Command Palette. You can use https://trace.playwright.dev/ to check the Playwright - https://trace.playwright.dev/.

The reasons could be that there is no focus on the editor's body or keyboard shortcuts must be corrected.

Reply:
Oh, interesting. Thanks for the tips. It's passing locally, so I'll try a few things and push them 😄

Reply:
Not sure if it helps, but it looks like the `Open the command palette and navigate to the page create page` test moves focus to the command palette button before pressing `Meta+k`, so something like that might make it more reliable? E.g. https://github.com/WordPress/gutenberg/blob/a3d81f6ff136d8676c84da79406ec58fc2fa3305/test/e2e/specs/site-editor/command-center.spec.js#L23-L25

Reply:
> but it looks like the Open the command palette and navigate to the page create page test moves focus to the command palette button before pressing Meta+k, so something like that might make it more reliable?

Thanks @andrewserong 

That was actually the first thing I tried (copy paste 😄 ) but it wasn't passing locally _with_ that line. 

Maybe I'll add it and YOLO it up to CI

Reply:
It's because that button only exists in view mode. I'm opening the canvas in edit mode. 👍🏻

Reply:
We could just click the button if we're already locating it 😄

Reply:
> We could just click the button if we're already locating it 😄 But as long as it works, I'm fine with it.

I ❤️ copy 🍝

Reply:
🍝 goes well with e2e tests 😁